cg/store/api/core.py

```diff
@@ -24,25 +21,13 @@ class CoreHandler(
     """Aggregating class for the store api handlers."""
 
     def __init__(self, session):
-        DeleteDataHandler(session=session)
-        FindBasicDataHandler(session=session)
-        FindBusinessDataHandler(session=session)
-        StatusHandler(session=session)
+        DeleteDataHandler(session)
+        FindBasicDataHandler(session)
+        FindBusinessDataHandler(session)
+        StatusHandler(session)
 
 
 class Store(CoreHandler):
-    uri: str = ""
-
-    def __init__(self, uri):
-        self.uri = uri
-        self.engine = create_engine(uri, pool_pre_ping=True)
-        self.session = scoped_session(sessionmaker(bind=self.engine))
-        super().__init__(session=self.session)
-
-    def create_all(self):
-        """Create all tables in the database."""
-        Model.metadata.create_all(bind=self.session.get_bind())
-
-    def drop_all(self):
-        """Drop all tables in the database."""
-        Model.metadata.drop_all(bind=self.session.get_bind())
+    def __init__(self):
+        self.session = get_session()

```

Comment:
This will be refactored at a later point. We are using a scoped session, it does not really matter how it is passed. I think it is confusing to have it as an attribute, but lets keep it for now.